### ISSUE TITLE
Where are annotations stored?

### ISSUE THREAD

**Issue by profucius** [xianmin/EveReader#9]
Are the annotations (e.g. highlights) stored inside the epub itself, and thus accessible by other apps? Or are they proprietary, or stored in a file managed by Eve Reader?

**Comment by xianmin** [comment]
No, not stored inside the epub itself. Now, it stored in indexedDB , and there is a simple import & export function. I will support more in the future.
Thank you for your feedback!
